src/main/java/cachiluis/model/Tutor.java

```diff
@@ -0,0 +1,4 @@
+package cachiluis.model;
+
+public class Tutor extends Person {
```

Comment:
Inheritors should add some functionality to parent classes. If not there are meaningless.